tools/cldr/cldr-to-icu/build-icu-data.xml

```diff
@@ -323,7 +323,7 @@
                     // F-P
                     fa_AF, fa, ff, fil, fi, fo, fr_BE, fr_CH, fr, ga, he, hi, hr,
                     hu, hy, id, in, is, it, iw, ja, ka, kl, km, ko, ky, lb,
-                    lo, lrc, lt, lv, mk, ms, mt, my, nb, nl, nn, no, pl, pt_PT, pt,
+                    lo, lrc, lt, lv, mk, ms, mt, my, nb, ne, nl, nn, no, pl, pt_PT, pt,
```

Comment:
Just curious, where does the "ne" locale come from? (And why only for RBNF?)

Reply:
It is not just for RBNF, it is in the regular localeId> list in this file, and has been a locale in ICU for some time now. The addition of RBNF data was per [CLDR-14177](https://unicode-org.atlassian.net/browse/CLDR-14177)

Reply:
Got it, thanks Peter!